src/charts.js

```diff
@@ -47,16 +47,61 @@ if (!Object.keys) {
   }());
 }
 
+function join(array, separator, lastSeparator) {
+  return array.slice(0, -1).join(separator) + lastSeparator + array.slice(-1)[0]
+}
+
 function getURLParam(name) {
   var match = new RegExp(name + '=' + '(.*?)(&|$)')
-      .exec(location.search);
+      .exec(decodeURIComponent(location.search));
   if (match === null)
     return null;
 
   return match[1];
 }
 
+var colors = ['#AA0000', '#00AA00', '#0000AA', '#AAAA00', '#00AAAA'];
 function chart(id, type, title, data, xAxisType, xAxisTitle, cats) {
+  var singleSerie = Object.keys(data).length === 1;
+  var series = [];
+  var i = 0;
+  for (var serieName in data) {
+    series.push({
+      name: serieName,
+      data: data[serieName],
+      color: colors[i]
+    });
+    ++i;
+  };
+
+  var growth = getURLParam('growth');
```

Comment:
Could be better. Maybe store a `dataType` param whose value can be either `downloads` (default) or `growth`, for now.

And this would then be:
```js
var dataType = getURLParam('dataType');
if (dataType === GROWTH_DATA) {
```